hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestColumnStatsIndexWithSQL.scala

```diff
@@ -204,33 +227,42 @@ class TestColumnStatsIndexWithSQL extends ColumnStatIndexTestBase {
     verifyFileIndexAndSQLQueries(commonOpts)
   }
 
-  private def setupTable(testCase: ColumnStatsTestCase, metadataOpts: Map[String, String], commonOpts: Map[String, String], shouldValidate: Boolean): Unit = {
+  private def setupTable(testCase: ColumnStatsTestCase, metadataOpts: Map[String, String], commonOpts: Map[String, String],
```

Comment:
can you add another simple test (even parametrize an existing test). where you set numColsToIndex to say 100. expected behavior should be same as not setting any explicit value for the config of interest.

Reply:
I think every test already does that since the default is 32. What we don't have though is testing if it is set to a low number and we specify the columns to index. I added that into a test